src/keyboard/textinput.js

```diff
@@ -512,7 +512,11 @@ TextInput= function(parentNode, host) {
         }
     };
 
-    event.addCommandKeyListener(text, host.onCommandKey.bind(host), host);
+    event.addCommandKeyListener(text, function(e, hashId, keyCode) {
+        if (inComposition && !hashId && (e.key == "Backspace" || e.key == "Delete"))
```

Comment:
does it make sense to handle `inComposition` case inside `onCommandKey`?

Reply:
i think no, because editor doesn't have direct access to inComposition, and weird browser specific behaviors are meant to be handled on textInput.js or when possible on event.js level

Reply:
how about `cancelComposition` on command keys?